src/pages/[platform]/deploy-and-host/sandbox-environments/features/index.mdx

```diff
@@ -112,32 +112,104 @@ npx amplify sandbox secret set foo --profile work
 
 You can create multiple cloud sandbox environments for each app if you want to keep persistent sandbox environments up and running to test against. First, run the following command in the terminal:
 
-```bash
+```bash showLineNumbers={false}
 npx amplify sandbox --name s1
 ```
 
 Once the deployment completes, exit sandbox `s1` and run the following command in the terminal:
 
-```bash
+```bash showLineNumbers={false}
 npx amplify sandbox --name s2
 ```
 
 After successful deployment, sandboxes `s1` and `s2` will be ready. Pick sandbox `s1` or `s2` to activate. You can switch between them but only one can be running at a time.
 
 ## Generate client config
 
-The client config, or `amplifyconfiguration.json` file, contains the configuration strings for interacting with AWS resources specific to an environment. The Amplify client libraries need the client config in order to use the library APIs to connect to backend resources. By default, the cloud sandbox generates the client configuration file at the root of the project (such as `@/amplifyconfiguration.json`). If you want to place the file at a different path (such as for a monorepo or Android app), run the following command in the terminal:
+The client config (e.g. `amplifyconfiguration.json` file for Web) contains the configuration strings for interacting with AWS resources specific to an environment. The Amplify client libraries need the client config file in order to use the library APIs to connect to backend resources. By default, the cloud sandbox generates the client configuration file at the root of the project (such as `@/amplifyconfiguration.json`). If you want to place the file at a different path you can use the following command template:
 
-```bash
-npx amplify sandbox --config-out-dir ./path/to/config --format ["mjs", "json", "json-mobile", "ts", "dart"]
+```bash showLineNumbers={false}
+npx amplify sandbox --config-out-dir=path/to/config --format=["mjs", "json", "json-mobile", "ts", "dart"]
```

Comment:
the equals sign is optional, but for consistency can we remove it from the command examples?